### PR TITLE
Update demo section in readme with steps for testing intuit#60

### PR DESCRIPTION
**Pull request by salilbc**
# Title
Update demo section in readme with steps for testing intuit#60

## Description of what's changing
Update demo section in readme with steps for testing intuit#60
...

## What else might be impacted?

...

## Which issue does this PR relate to?

...

## Checklist

[ ] Tests are written and maintain or improve code coverage
[ ] I've tested this in my application using `yarn link` (if applicable)
[ ] You consent to and are confident this change can be released with no regression

### CODE REVIEW

**Comment by reubenae** [comment]
Will take a look later but I suggest in future you check out a specific feature branch for each PR - it would mean there wouldn't be 27 commits for a doc change

File: src/js/common/browserInfoHelper.js [hunk 1]
```diff
@@ -105,4 +105,4 @@ export const getBrowserDoNotTrackStatus = () => {
       windowVar.external.msTrackingProtectionEnabled &&
       windowVar.external.msTrackingProtectionEnabled());
   return isBrowserDoNotTrack ? "true" : "false";
-};
+};
```
Comment (reubenae) [inline src/js/common/browserInfoHelper.js:108]: This file shouldn't have changed - please checkout the same version from master - here is a guide http://nicolasgallagher.com/git-checkout-specific-files-from-another-branch/

File: tests/unit/hmrc/mtdFraudPrevention.test.js [hunk 2]
```diff
@@ -100,4 +100,4 @@ describe("FraudPreventionHeaders", () => {
     expect(headers.get("Gov-Client-Device-ID")).toEqual("fce4f7ff-d5f1-4e4f-99a1-aa97bef71e99");
     expect(errors[0]).toEqual("Something went wrong.");
   });
-});
+});
```
Comment (reubenae) [inline tests/unit/hmrc/mtdFraudPrevention.test.js:103]: This file shouldn't have changed - please checkout the same version from master - here is a guide http://nicolasgallagher.com/git-checkout-specific-files-from-another-branch/

**Review by reubenae — CHANGES_REQUESTED** [review]

**Comment by salilbc** [comment]
Have cleaned this up based  on your recommendation. Please refer new PR https://github.com/intuit/user-data-for-fraud-prevention/pull/67

File: src/js/common/browserInfoHelper.js (hunk 1 above)
Comment (salilbc) [reply src/js/common/browserInfoHelper.js:108]: Cleaned. this up, refer new PR https://github.com/intuit/user-data-for-fraud-prevention/pull/67

File: tests/unit/hmrc/mtdFraudPrevention.test.js (hunk 2 above)
Comment (salilbc) [reply tests/unit/hmrc/mtdFraudPrevention.test.js:103]: Cleaned. this up, refer new PR https://github.com/intuit/user-data-for-fraud-prevention/pull/67

**Pull request closed without merging**